RELEASE_NOTES.md

```diff
@@ -4,26 +4,135 @@
 
 #### Chef/UseInlineResourcesDefined
 
+The `UseInlineResourcesDefined` checks for resources that call the `use_inline_resources` method. Chef Infra Client 13 made the inline resources mode the default for all resources, so this call is no longer necessary.
+
+`Enabled by default`: True
+
+`Autocorrects`: Yes
+
 #### Chef/IncludingOhaiDefaultRecipe
 
+The `IncludingOhaiDefaultRecipe` detects cookbooks that include the `ohai::default` recipe. The ohai cookbook's default recipe was used to install custom Ohai plugins located in the cookbook's `files` directory. This required forking the ohai community cookbook to add your own plugins, which we no longer recommend doing. The Ohai cookbook provides an [ohai_plugin resource](https://github.com/chef-cookbooks/ohai/#ohai_plugin) which can be used to install plugins and Chef Infra Client. Chef Infra Client. Additionally plugins can be placed in a `/ohai` directory in any cookbook to have Chef Infra Client install install these at runtime. Note: This feature requires Chef Infra *Server* 12.18 or later.
```

Comment:
One more thing - please clarify what "This" is per the context of "This required forking the ohai community cookbook" Thanks!

Reply:
Fixed